packages/devextreme/js/__internal/ui/html_editor/utils/ai.ts

```diff
@@ -0,0 +1,106 @@
+import type {
+  AIChangeStyleOption,
+  AIChangeToneOption,
+  AICommandName,
+  AIToolbarItem,
+  AITranslateOption,
+} from '@js/ui/html_editor';
+import { capitalize } from '@ts/core/utils/capitalize';
+
+type CommandOption =
+  | AIChangeStyleOption
+  | AIChangeToneOption
+  | AITranslateOption;
+
+export interface CommandDefinition {
+  name: string;
+  text: string;
+  options?: (CommandOption | string)[];
+}
+
+export type CommandsMap = Record<string, CommandDefinition>;
+
+export const defaultCommandNames: Record<AICommandName, string> = {
+  summarize: 'Summarize',
+  proofread: 'Proofread',
+  expand: 'Expand',
+  shorten: 'Shorten',
+  changeStyle: 'Change Style',
+  changeTone: 'Change Tone',
+  translate: 'Translate',
+  askAI: 'Ask AI',
+};
+
+const htmlEditorAIChangeStyleOptions: AIChangeStyleOption[] = [
+  'formal', 'informal', 'technical', 'business',
+  'creative', 'journalistic', 'academic', 'persuasive',
+  'narrative', 'expository', 'descriptive', 'conversational',
+];
+
+const htmlEditorAIChangeToneOptions: AIChangeToneOption[] = [
+  'professional', 'casual', 'straightforward', 'confident', 'friendly',
+];
+
+const htmlEditorAITranslateOptions: AITranslateOption[] = [
+  'arabic', 'chinese', 'english', 'french', 'german', 'japanese', 'spanish',
+];
+
+export const getDefaultOptionsByCommand = (
```

Comment:
✏️ **[code style]**: more compact and easy to read

```suggestion
export const getDefaultOptionsByCommand = (command: AICommandName | 'custom'): CommandOption[] | undefined => {
  const options: Record<string, CommandOption[]> = {
    changeStyle: htmlEditorAIChangeStyleOptions,
    changeTone: htmlEditorAIChangeToneOptions,
    translate: htmlEditorAITranslateOptions,
  };

  return options[command];
};
```